src/frontend/package.json

```diff
@@ -40,6 +40,7 @@
     "@typescript-eslint/parser": "8.23.0",
     "docx": "9.1.1",
     "eslint": "8.57.0",
+    "prosemirror-view": "1.37.2",
```

Comment:
Do we have to set a rule in renovate configuration ? Like ignoring it because we want to manage the version our self ?

Reply:
We could but Blocknote made a fix for these 2 already:
- https://github.com/TypeCellOS/BlockNote/pull/1494
- https://github.com/TypeCellOS/BlockNote/pull/1431
Docx seems to work with newer version as well.

I have already a PR opened about bumping Blocknote, I think it will be ready before renovate tried to bump them.